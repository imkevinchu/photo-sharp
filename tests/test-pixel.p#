preset int main():
{
  Pixel p

  p = (12, 3, 4, 1)

  print(p)

  return 0
}
