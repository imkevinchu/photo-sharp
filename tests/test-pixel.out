Red: 12
Green: 3
Blue: 4
Alpha: 1
